index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="styles/header.css">
+  <link rel="stylesheet" href="styles/headline.css">
+  <link rel="stylesheet" href="styles/general.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header class="header">
+    <nav class="nav">
+      <div class="left-section">
+        <p class="logo">My Logo</p>
+      </div>
+      <button class="right-section">
+        <img class="icon-menu" src="/Normal Button/Tertiary/Icons/Icon - Menu.png" alt="">
+      </button>
+    </nav>
+    <section class="headline">
+      <h2 class="introduction">I’m Mariane
+        Glad to see you!</h2>
```

Comment:
- Kindly make sure The main title (e.g., "I'm Marianne ...") is an h1 tag. so I suggest you use `h1 ` instead of `h2`.  :+1: 

If you want that google (and other search engines) know about a specific topic, in this case, you, use an h1 tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.